timeseries/src/autogluon/timeseries/models/gluonts/torch/models.py

```diff
@@ -0,0 +1,181 @@
+
+        trainer_kwargs = {}
+        epochs = init_args.get("max_epochs", init_args.get("epochs"))
+        callbacks = init_args.get("callbacks", [])
+
+        if epochs is not None:
+            trainer_kwargs.update({"max_epochs": epochs})
+        trainer_kwargs.update({"callbacks": callbacks, "enable_progress_bar": False})
+
+        return from_hyperparameters(
+            self.gluonts_estimator_class,
+            trainer_kwargs=trainer_kwargs,
+            **init_args,
+        )
+
+    def _fit(
+        self,
+        train_data: TimeSeriesDataFrame,
+        val_data: Optional[TimeSeriesDataFrame] = None,
+        time_limit: int = None,
+        **kwargs,
+    ) -> None:
+        verbosity = kwargs.get("verbosity", 2)
+        set_logger_verbosity(verbosity, logger=logger)
+        gts_logger.setLevel(logging.ERROR if verbosity <= 3 else logging.INFO)
+        for pl_logger in pl_loggers:
+            pl_logger.setLevel(logging.ERROR if verbosity <= 3 else logging.INFO)
+
+        if verbosity > 3:
+            logger.warning(
+                "GluonTS logging is turned on during training. Note that losses reported by GluonTS "
+                "may not correspond to those specified via `eval_metric`."
+            )
+
+        self._check_fit_params()
+        # TODO: reintroduce early stopping callbacks
+
+        # update auxiliary parameters
+        callbacks = [Timer(timedelta(seconds=time_limit))] if time_limit is not None else []
+        self._deferred_init_params_aux(dataset=train_data, callbacks=callbacks, **kwargs)
+
+        estimator = self._get_estimator()
+        with warning_filter(), disable_root_logger():
+            self.gts_predictor = estimator.train(
+                self._to_gluonts_dataset(train_data),
+                validation_data=self._to_gluonts_dataset(val_data),
+            )
+
+    @classmethod
+    def load(cls, path: str, reset_paths: bool = True, verbose: bool = True) -> "AbstractGluonTSModel":
+        model = super().load(path, reset_paths, verbose)
+        model.gts_predictor = GluonTSPyTorchPredictor.deserialize(Path(path) / cls.gluonts_model_path)
+        return model
+
+
+class DeepARModel(AbstractGluonTSPyTorchModel):
+    gluonts_estimator_class: Type[GluonTSPyTorchLightningEstimator] = DeepAREstimator
+
+
+class SimpleFeedForwardModel(AbstractGluonTSPyTorchModel):
+    gluonts_estimator_class: Type[GluonTSPyTorchLightningEstimator] = SimpleFeedForwardEstimator
+
+    def _get_estimator_init_args(self) -> Dict[str, Any]:
+        init_kwargs = super()._get_estimator_init_args()
+
+        # FIXME: PyTorch StudentT does not implement quantile functions
+        if "distr_output" in init_kwargs:
+            warnings.warn(
+                f"distr_output {init_kwargs['distr_output']} specified for SimpleFeedForward, however training"
+                "will default to the Gaussian distribution.",
+                category=UserWarning,
```

Comment:
No need to specify this, `UserWarning` is the default https://docs.python.org/3/library/warnings.html#warnings.warn